test/integration/cdi/example-driver/deploy/example/pod-external.yaml

```diff
@@ -0,0 +1,41 @@
+# One external resource claim, one pod, two containers
+# One container uses resource, one does not
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: external-claim-parameters
+  namespace: default
+data:
+  a: b
+---
+apiVersion: v1
+kind: ResourceClaim
+metadata:
+  name: external-claim
+spec:
+  resourceClassName: example
+  parameters:
+    apiVersion: v1
+    kind: ConfigMap
+    name: external-claim-parameters
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-external-claim
+spec:
+  restartPolicy: Never
+  containers:
+  - name: with-resource
+    image: ubuntu:22.04
```

Comment:
Pulling from Docker Hub is problematic. Can we use an image that is already used by Kubernetes E2E testing and hosted on gcr?

Reply:
done

Reply:
@pohly btw, we can use smaller image, e.g. gcr.io/google-containers/busybox. Let me know if you think it's better.

Reply:
Let's use a busybox image that is already a dependency of E2E testing:
https://github.com/kubernetes/kubernetes/blob/305ad47627d7cc67d90389f5f8c27c37ebe4bf3a/test/utils/image/manifest.go#L240

-> registry.k8s.io/e2e-test-images/1.29-2

That will make it easier later on to integrate this with E2E testing.

Reply:
done, please review again